engine/src/main/java/org/hibernate/validator/internal/constraintvalidators/hv/ScriptAssertValidator.java

```diff
@@ -6,37 +6,61 @@
  */
 package org.hibernate.validator.internal.constraintvalidators.hv;
 
+import static org.hibernate.validator.internal.util.logging.Messages.MESSAGES;
+
 import javax.validation.ConstraintValidator;
 import javax.validation.ConstraintValidatorContext;
 
 import org.hibernate.validator.constraints.ScriptAssert;
 import org.hibernate.validator.internal.util.Contracts;
-
-import static org.hibernate.validator.internal.util.logging.Messages.MESSAGES;
+import org.hibernate.validator.internal.util.logging.Log;
+import org.hibernate.validator.internal.util.logging.LoggerFactory;
 
 /**
  * Validator for the {@link ScriptAssert} constraint annotation.
  *
  * @author Gunnar Morling
  * @author Hardy Ferentschik
  * @author Kevin Pollet &lt;kevin.pollet@serli.com&gt; (C) 2011 SERLI
+ * @author Marko Bekhta
  */
 public class ScriptAssertValidator implements ConstraintValidator<ScriptAssert, Object> {
 
+	private static final Log log = LoggerFactory.make();
```

Comment:
log=LOG (I know we have it lowercase in many places, but in new code we tend to use the proper casing for constants).